### PR TITLE
Medias Digimon

### PR DESCRIPTION
**Pull request by davidTocineta**

### CODE REVIEW

File: 06/index.html [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        <img src="./assets/Gomamon.webp" width="125">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+        <li>Zudomon</li>
+        <img src="./assets/Zudomon.webp" width="150">
+        <audio src="./assets/videoplayback.mp4" controls></audio>
+      </ol>
+    </nav>
+    <aside>
+      <h2>Opening</h2>
+      <video src="./assets/Digimon Adventure 01 Opening Español Latino (720p).mp4" controls poster="./assets/digimon miniatura.jpeg"></video>
```
Comment (eamzea) [inline 06/index.html:59]: This is not the best way of naming a file, remember that almost everything is written in English so the `ñ` does not exist.
Esa no es la mejor forma de nombrar un archivo, recuerda que casi todo está escrito en inglés por lo que la `ñ` no existe.

Remember about casing -> PASCAL, SNAKE, CAMEL, KEBAB

File: 06/index.html [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      <h2>Demo</h2>
+      <iframe src="https://digimon.fandom.com/es/wiki/Digimon_Adventure" width="900" height="600"></iframe>
+    </article>
+    <section>
+      <from>
+        <h3>Acerca de ti</h3>
+        <fieldset>
+          <span>Nombre:</span>
+          <br>
+          <input type="text" value="Nombre">
```
Comment (eamzea) [inline 06/index.html:71]: Instead of span what tag you can use?

Comment (eamzea) [inline 06/index.html:71]: There's something missing in this input

File: 06/index.html [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+            <input type="radio" id="Si" name="decision">
+            <br>
+            <label for="No">No</label>
+            <input type="radio" id="No" name="decision">
+          </fieldset>
+          <br>
+          <label for="digimon">Selecióna tu pokemon favorito</label>
+          <br>
+          <select id="digimon" name="digimon">
+            <option>--Selecciona un pokemon--</option>
```
Comment (eamzea) [inline 06/index.html:86]: This option is still selectable, that is the correct behavior?

File: 06/index.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+            <option value="Togemon">Togemon</option>
+            <option value="WereGarurumon">WereGarurumon</option>
+            <option value="Zudomon">Zudomon</option>
+          </select>
+          <br>
+          <br>
+          <fieldset>
+            <legend>Terminos</legend>
+            <label>He leído los datos</label>
+            <input type="checkbox">
```
Comment (eamzea) [inline 06/index.html:103]: What is missing in this label and input?

File: 06/index.html [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+          </fieldset>
+          <br>
+          <input type="submit" value="Enviar encuesta">
```
Comment (eamzea) [inline 06/index.html:106]: Another missing value in this input

File: 06/index.html
Comment (eamzea) [inline 06/index.html:1]: Every image input is missing something important

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

File: 06/index.html (hunk 4 above)
Comment (davidTocineta) [reply 06/index.html:103]: For and id

File: 06/index.html [hunk 6]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="author" content="Jesus David Ceballos Parra">
+  <meta name="description" content="Informacion sobre la serie de digimon y algunas imagenes y audios">
+  <meta name="keywords" content="Digimon, Series, Anime, Adventuras, Monstrous, Niños Elegidos">
+  <title>Tarea con digimons</title>
```
Comment (eamzea) [inline 06/index.html:8]: This is not properly aligned

File: 06/index.html [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+    <aside>
+      <h2>Opening</h2>
+      <video src="./assets/Digimon-adventure-Opening-latino.mp4" controls poster="./assets/digimon miniatura.jpeg"></video>
+    </aside>
+    <article>
+      <h2>Demo</h2>
+      <iframe src="https://digimon.fandom.com/es/wiki/Digimon_Adventure" width="900" height="600"></iframe>
+    </article>
+    <section>
+      <from>
```
Comment (eamzea) [inline 06/index.html:66]: What is this tag?

**Review by soyedgarzea — CHANGES_REQUESTED** [review]

**Review by soyedgarzea — APPROVED** [review]

**Pull request merged**